Source/WebCore/platform/network/mac/UTIUtilities.mm

```diff
@@ -47,9 +48,8 @@
 
 String MIMETypeFromUTI(const String& uti)
 {
-ALLOW_DEPRECATED_DECLARATIONS_BEGIN
-    return adoptCF(UTTypeCopyPreferredTagWithClass(uti.createCFString().get(), kUTTagClassMIMEType)).get();
-ALLOW_DEPRECATED_DECLARATIONS_END
+    auto *type = [UTType typeWithIdentifier:uti];
```

Comment:
```suggestion
    RetainPtr type = [UTType typeWithIdentifier:uti];
```

(since you're changing this line anyway, and we have explicit smart pointer guidelines now)